utils/databake/src/primitives.rs

```diff
@@ -121,15 +121,14 @@ fn byte_string(bytes: &[u8]) -> TokenStream {
     // escape slash when trying to match the first "\0" with another digit, and then seeing b"\01".
```

Comment:
I'm inclined to remove this, as our MSRV is not past 1.70, and this has apparently been generating iffy code (attribute on expression), with nobody complaining. wdyt?

Reply:
Whatever makes the nicest output and fewest warnings 🤷